core/frontend/src/Viewport.ts

```diff
@@ -1334,6 +1334,7 @@ export abstract class Viewport implements Disposable, TileUser {
       removals.push(settings.onAmbientOcclusionSettingsChanged.addListener(displayStyleChanged));
       removals.push(settings.onEnvironmentChanged.addListener(displayStyleChanged));
       removals.push(settings.onPlanProjectionSettingsChanged.addListener(displayStyleChanged));
+      removals.push(settings.onContoursChanged.addListener(displayStyleChanged));
```

Comment:
@markschlosseratbentley my tests weren't working because this line was missing. After this PR completes please make sure someone adds some proper basic unit tests.

Reply:
I filed an item: https://github.com/iTwin/itwin-graphics-backlog/issues/655